### ISSUE TITLE
About reproducible result for s3dis dataset

### ISSUE THREAD

**Issue by amiltonwong** [pqhieu/sota3d#2]
Hi, @pqhieu ,

I had followed the training steps on s3dis dataset using the default pointnet.yml and pointnet2.yml. However, I cannot obtain the reported IoU value (Pointnet: `0.496 ± 0.120` , Pointnet2: `0.580 ± 0.100`). I only got these lower IoU value (Pointnet: `0.439` , Pointnet2: `0.539`)

Is there any other config to achieve the reported IoU value?

THX!

**Comment by pqhieu** [comment]
Hi @amiltonwong,

Have you tried k-fold validation?  The results that I got is from 6-fold validation by holding out different area as the test set. You can do it by invoking the following command:
```bash
$ python -m sota3d.s3dis.main --config configs/s3dis/<model>.yml --dataset.test_area {1, ..., 6}
```

**Comment by amiltonwong** [comment]
Hi, @pqhieu ,

Thanks for your reply. Could you elaborate the specific commands?
I had tried these two commands, but it returns error:
`python -m sota3d.s3dis.main --config configs/s3dis/pointnet2.yml --dataset.test_area {1, ..., 6}`
`python -m sota3d.s3dis.main --config configs/s3dis/pointnet2.yml --dataset.test_area {1, 2, 3, 4, 5, 6}`.

Did you mean to run the following six commands, one by one?
```
python -m sota3d.s3dis.main --config configs/s3dis/pointnet2.yml --dataset.test_area 1
python -m sota3d.s3dis.main --config configs/s3dis/pointnet2.yml --dataset.test_area 2
python -m sota3d.s3dis.main --config configs/s3dis/pointnet2.yml --dataset.test_area 3
python -m sota3d.s3dis.main --config configs/s3dis/pointnet2.yml --dataset.test_area 4
python -m sota3d.s3dis.main --config configs/s3dis/pointnet2.yml --dataset.test_area 5
python -m sota3d.s3dis.main --config configs/s3dis/pointnet2.yml --dataset.test_area 6
```

**Comment by pqhieu** [comment]
Yes, that's correct.

**Comment by amiltonwong** [comment]
Hi, @pqhieu ,

Here, I updated pointnet2 iou records for these six areas as follows:
area1 | 0.643
area2 | 0.407
area3 | 0.679
area4 | 0.504
area5 | 0.524
area6 | 0.711

And the average iou value across six areas is `0.578`. It shows consistent result as reported. BTW, how did you select the model among six models after 6-fold cross validation? Did you choose the one with the highest test iou value (e.g. the one trained on area 1,2,3,4,5, and tested on area 6)?

**Comment by pqhieu** [comment]
This is for benchmarking so I took the mean values across different folds. Some papers only reported the results on area 5, which is the recommended split.
